q2templates/templates/assets/css/base-template.css

```diff
@@ -0,0 +1,25 @@
+/*! normalize.css v5.0.0 */
+
+/**
+ * 1. Change the width and margin values for table populated in base.html.
+ * 2. Center the data cells and table headers.
+ */
+
+/**
+* Center table heads in each column and related text in table.
+* Center table in page with reduced size.
+*/
+   
+div.text-center {
+  margin-left: 10%;
+  justify-content: center;
+  width: 80%;
```

Comment:
This makes me a little bit nervous, as the CSS selector looks relatively generic. Could you elaborate on the purpose of the percentage values (and where the `text-center` class generally appears)?